shared-components/Form/TimeRangeSelect.tsx

```diff
@@ -58,14 +58,14 @@ export default function TimeRangeSelect() {
       setValue("from", date2String(from));
     }
 
-    if (toFormValue && toFormValue.getDate() !== to.getDate()) {
+    if (toFormValue && new Date(toFormValue).getDate() !== to.getDate()) {
```

Comment:
Because the DatePicker want's a string and then returns a string. In Statviz this was not an issue as type checking was not that strickt so i could pass a date object into the DatePicker and it worked.

Maybe the DatePicker component should be adjusted but I didn't want to spend to much time checking this in Detail